homeworks/lesia.turua_lesiaturuta/PokemonProject/about.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>PokemonProject</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="./style.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <nav class="nav justify__content--center">
+                <div class="container justify__content--between">
+                    <div class="nav__logo">
+                        <img class="logo__icon" src="./assets/logo-icon.svg" alt="logo">
+                        <img class="logo__title" src="./assets/name-icon.svg" alt="name">
+                    </div>
+                    <ul class="nav__menu">
+                        <li>
+                            <a class="menu__link" href="./cards.html">Pokemons</a>
+                        </li>
+                        <li>
+                            <a class="menu__link menu__link--active" href="./about.html">About</a>
+                        </li>
+                    </ul>
+
+                </div>
+            </nav>
+
+        </header>
+        <main class="main">
+            <div class="container justify__content--between">
+                <div class="main__image">
+                    <img src="./assets/pokemon.png" alt="pokemon">
+                </div>
+                <article class="info">
+                    <h1 class="description">About me</h1>
+                    <p class="text">Hi! My name is [Text] and I'm a Junior Frontend Developer. I am already familiar with
+                        main
+                        Web
+                        Technologies like HTML, CSS, JavaScript, and Git version control system.</p>
+                    <p class="text">This page was developed during the course
+                        <a class="link--red" href="https://www.mastersacademy.education/frontend-for-beginners-it"
+                            target="_blank">'Frontend for beginners'
+                            from Masters Academy in 2023.</a>
+                    </p>
+                    <p class="text">This is a social project from MOCG company where I got an opportunity to work with
+                        Frontend
+                        mentors and to create my own small project for the portfolio.</p>
+                    <p class="text">You can contact me via [Links to Socials Nets] and/or check out my GitHub.</p>
```

Comment:
pls add your social nets and git hub links

Reply:
added links